benchopt/runner.py

```diff
@@ -254,7 +255,12 @@ def __call__(self, x):
                 return False
             if self.it == self.max_iter:
                 return False
+            current_obj = objective_dict["objective_value"]
+            if (-EPS <= self.prev_objective - current_obj <= EPS):
+                self.status = 'done'
+                return False
 
+            self.prev_objective = current_obj
```

Comment:
is prev_objective in the class docstring?

Reply:
It wasn't but it is now